backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -18,7 +20,17 @@
 
 	public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+	public TestEvent findFirst1ByOrganizationAndInternalId(Organization o, UUID id);
+
 	// Need to control how this query is built. "between" is too vague.
+	// This is across all Orgs/facilities because datahub uploader users
 	@Query("FROM #{#entityName} q WHERE q.createdAt > :before AND q.createdAt <= :after ORDER BY q.createdAt DESC ")
 	public List<TestEvent> queryMatchAllBetweenDates(Date before, Date after);
+
+	@Query( value = "SELECT DISTINCT ON (test_order_id) * " +
+					" FROM {h-schema}test_event te " +
+					" WHERE te.facility_id = :facilityId " +
+					" AND te.created_at > :newerThanDate" +
+					" ORDER BY te.test_order_id, te.created_at desc", nativeQuery = true)
+	public List<TestEvent> getTestEventResults(UUID facilityId, Date newerThanDate);
```

Comment:
We might be able to make this a Facility instead of a UUID with a little SpEL shenanigans. Later.